### DISCUSSION TITLE
Axhttp Questions

### DISCUSSION THREAD

**Discussion by ivdev01** [AxisCommunications/acap-native-sdk-examples#102] [Q&A] [answered]
Hello,

I have an application built using ACAP2 which used net_http library which I'm trying to update to work with the newer cameras. I've been trying to update to ACAP3 and using Axhttp to get try to get my CGI's working. I'm following the documentation the best that I can but when I install the application and I try to go to a CGI's location it return path not found. It does not seem to even go in to the request_handler function in my code at all.

I've trying to find some examples online to see if I'm missing some configuration but no luck and if anyone could point at the right direction or if anyone has any insight on this issue it would be greatly appreciated.

Thanks.

PS I moved this question to the general discussion for more potential visibility.

**Comment by pataxis** [comment, marked as answer]
Hi @ivdev01 ,
it seems you found what was the issue in https://github.com/orgs/AxisCommunications/discussions/152.
Let us know otherwise.
